Author: Denis Briand <debian@denis-briand.fr>
Bug-Debian: https://bugs.debian.org/859407
Description: if USER env variable isn't defined, use nobody account to avoid SIGSEGV with getpwnam

--- a/filter.c
+++ b/filter.c
@@ -186,6 +186,11 @@
 	int rtn;
 	char *tmp;
 
+	//if USER env variable isn't defined, use nobody account to avoid SIGSEGV with getpwnam
+	if (!username) {
+          username = "nobody";
+       }
+
 	pwent = getpwnam(username);
 
 	if((tmp = xstrdup(pwent->pw_gecos)) == NULL)
